packages/ui-kit/package.json

```diff
@@ -17,7 +17,7 @@
   "module": "dist/commercetools-docs-ui-kit.esm.js",
   "files": ["dist", "package.json", "LICENSE", "README.md"],
   "scripts": {
-    "generate-icons": "svgr -d src/icons/generated -- src/icons/svg"
+    "generate-icons": "svgr -d src/icons/generated --typescript --filename-case kebab --no-index -- src/icons/svg"
```

Comment:
The CamelCasing used before made eslint very unhappy when importing them where they are used.  Rabbithole: with kebab case generated components the automatic generation of the index file fails

Reply:
I'm not sure I understand the issue here...

Reply:
resolved through allowing camelCased typescript imports